operations/services.md

```diff
@@ -214,6 +216,134 @@ Updates the number of available resources in [Resource category](enterprises.md#
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `AvailabilityBlocks` | array of [Availability block parameters](#availability-block-parameters) | required, max 1000 items | Availability blocks to be added. |
+
+#### Availability block parameters
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `ServiceId` | string | required | Unique identifier of the [Service](#service) to assign block to. |
+| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `ReleasedUtc` | string | required | Date when the [Availability update](#availability-update)s linked to the block will not reduce public availability in UTC timezone in ISO 8601 format. |
+
+### Response
+
+```javascript
+{
+    "AvailabilityBlocks": [
+        {
+            "Id": "e5a4654a4a94-86da-4f96-9efc-bd26d8db",
+            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+            "StartUtc": "2020-10-05T00:00:00Z",
+            "EndUtc": "2020-10-06T00:00:00Z",
+            "ReleasedUtc": "2020-10-04T00:00:00Z",
+        },
+        {
+            "Id": "aaaa654a4a94-4f96-9efc-86da-bd26d8db",
+            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
+            "StartUtc": "2020-11-05T00:00:00Z",
+            "EndUtc": "2020-11-06T00:00:00Z",
+            "ReleasedUtc": "2020-11-04T00:00:00Z",
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `AvailabilityBlocks` | array of [Availability blocks](#availability-block) | required | Availability blocks. |
+
+#### Availability block
+
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the availability block. |
+| `ServiceId` | string | required | Unique identifier of the [Service](#service) the block is assigned to. |
+| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `ReleasedUtc` | string | required | Date when the [Availability update](#availability-update)s linked to the block will not reduce public availability in UTC timezone in ISO 8601 format. |
+
+## Delete availability blocks
+
+Delete availability blocks. Only block with no active [Availability update](#availability-update)s and [Reservation](#reservation)s can be deleted
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/availabilityBlocks/delete`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "AvailabilityBlocks": [
```

Comment:
Let's turn this just into Ids as I commented in the code PR, forcing partners to provide redundant service ids only because of us doesn't make sense. Service Id is not needed here, we shouldn't require it.

Reply:
But still somewhere has to be cross checked that you are deleting block from their enterprise not someone other and not from other service if it per service scope

Reply:
Correct, that has to be in the api - you first fetch blocks, load services and then check if you can modify the blocks. Actually, even without checking in API, the privileges should capture this and not allow deleting. But I think we should be checking in API. Just make sure that even without checks, it would fail on privielges.

Reply:
Well by the implementation we have enterprise and we get its services and by these services we can get blocks so in combination you wont get block because service is not the one from same hotel so it will fail on invalid block id.

Reply:
👌 so Ids are sufficient here. and in the API we will load all services in the hotel and then load blocks by services and Ids.